src/CommunityToolkit/Xamarin.CommunityToolkit.Sample.Android/Properties/AndroidManifest.xml

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="1" android:versionName="1.0" package="com.companyname.xamarincommunitytoolkitsample" android:installLocation="auto">
-	<uses-sdk android:minSdkVersion="21" android:targetSdkVersion="28" />
+	<uses-sdk android:minSdkVersion="21" android:targetSdkVersion="29" />
```

Comment:
can we update to Android 11 and use sdk version 30?

Reply:
I don't want to change the samples on this PR.